cmd/examples/server.cpp

```diff
@@ -102,6 +102,12 @@ class MySubscribeTrackHandler : public quicr::SubscribeTrackHandler
 
     void ObjectReceived(const quicr::ObjectHeaders& object_headers, quicr::BytesSpan data) override
     {
+        if (data.size() > 255) {
+            SPDLOG_CRITICAL("Example server is for example only, received data > 255 bytes is not allowed!");
+            SPDLOG_CRITICAL("Use github.com/quicr/laps for full relay functionality");
+            throw std::runtime_error("Example server is for example only, received data > 255 bytes is not allowed!");
+        }
```

Comment:
do we see any issue with allowing object sizes greater than 256 with qserver ?

Reply:
The idea is that example server should be examples only, such as simple text.  It should be less than 255 bytes in this case. You will not be able to send video because this is not intended for that.  LAPS is developed to handle video and other real-time use-cases. This is just an example of how to use the library.